content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -695,6 +696,8 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `RotateKubeletServerCertificate`: Enable the rotation of the server TLS certificate on the kubelet.
   See [kubelet configuration](/docs/reference/access-authn-authz/kubelet-tls-bootstrapping/#kubelet-configuration)
   for more details.
+- `RuntimeClassInImageCriApi` : Enables images to be pulled based on the [runtime class](https://kubernetes.io/docs/concepts/containers/runtime-class/) of the Pods
+  that reference them.
```

Comment:
We make use of site-relative links whenever feasible and also, ensure that long text lines are wrapped.

```suggestion
- `RuntimeClassInImageCriApi`: Enables images to be pulled based on the [runtime class](/docs/concepts/containers/runtime-class/)
of the Pods that reference them.
```